jenkins/custom-ci/generic/hare.groovy

```diff
@@ -98,7 +106,12 @@ EOF
                 script { build_stage = env.STAGE_NAME }
                 sh label: 'Copy RPMS', script: '''
                     mkdir -p $build_upload_dir
-                    cp /root/rpmbuild/RPMS/x86_64/*.rpm $build_upload_dir
+                    if [ "$RECREATE_HARE_RPM" == "yes" ]; then
+                            cp /root/rpmbuild/RPMS/x86_64/*.rpm $build_upload_dir
+                    else
+                        echo "Copy packages form last_successful"
+                        cp /mnt/bigstorage/releases/cortx/components/github/main/rockylinux-8.4/dev/hare/last_successful/*.rpm $build_upload_dir
```

Comment:
Please use `$os_version/$release_tag` instead of hardcoded values

Reply:
as we will be using the last successful build hence using the hardcode path to avoid any conflict 
Refer  : https://github.com/Seagate/cortx-re/blob/a45a76580587176f55b1e99acb736be7e0b2650e/jenkins/custom-ci/generic/cortx-rgw.groovy#L81